administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss

```diff
@@ -1,6 +1,6 @@
 // Custom Forms
 
-.form-select {
+.custom-select {
```

Comment:
I'd done this deliberately because of the var changes and thought that combined with this alias I would have things working :/ https://github.com/joomla/joomla-cms/pull/32037/files#diff-f81e8b4ffce389a8b743a7a2be34106f05ba6de248f9342471733c75cf919b7eR180 

Why do we need this one?

Reply:
Yep you are right. I'd forgotten this was a bootstrap class. It seems `custom-select` along with our own custom variants of this (eg. `custom-select-color-state`) was still been set in all the xmls and a number of views. A blanket search and replace to `form-select-*` seems to be working fine. I'll push it once I test a little further.

Reply:
Yeah I thought the `@extend` would work to cover the majority of cases (at least in admin I think the native custom-select works - I'm just unsure about the nested properties)